idr0045-study.txt

```diff
@@ -0,0 +1,54 @@
+Experiment Size		Average Image Dimension (XYZCT): 1000x1000x101x2x[120-181] Total Gb: 259	Average Image Dimension (XYZCT): [576-640]x[576-640]x[47-62]x4x1 Total Mb: 840	Average Image Dimension (XYZCT): [596-688]x[596-688]x[73-130]x4x1 Total Gb: 8.0	Average Image Dimension (XYZCT): 1000x1000x101x3x[130-260]  Total Gb: 414	Average Image Dimension (XYZCT): 1000x1000x101x2x81 Total Gb: 244	Average Image Dimension (XYZCT): [200-250]x[200-250]x[45-101]x2x1 Total Gb: 1.0	Average Image Dimension (XYZCT): [250-340]x[250-340]x[45-100]x3x[20-90] Total Gb: 12	Average Image Dimension (XYZCT): 576x576x[71-149]x4x8 Total Gb: 8.7	Average Image Dimension (XYZCT): [332-500]x[357-500]x5x1x[45-100] Total Gb: 0.8	Average Image Dimension (XYZCT): [200-250]x[200-250]x[35-100]x3x1 Total Gb: 0.8	Average Image Dimension (XYZCT): 600x600x[130-150]x2x1 Total Gb: 8	Average Image Dimension (XYZCT): 1000x1000x101x3x[14-135] Total Gb: 324	Average Image Dimension (XYZCT): 448x448x[40-107]x3x1 Total Gb: 9.7	Average Image Dimension (XYZCT): 448x448x[48-125]x2x1 Total Gb: 3.9	Average Image Dimension (XYZCT): 448x448x[46-116]x2x1  Total Gb: 6.5
+Experiment Example Images		Embryo3	MicrotubuleBallStage4, Prometaphase1, Prometaphase3, Metaphase4	PM10, PM15	Embryo05						Zygote/Cell03, 2Cell/2-Cell-10-1, 4Cell/4-Cell-11-1 and 8Cell/8-Cell-14-1		NocMo/Nocodazole/Embryo2, NocMo/Monastrol/Embryo3, NocMo/Washout/Embryo1, MoNoc/Monastrol/Embryo1, MoNoc/NocodazoleEmbryo1 and MoNoc/Washout/Embryo1	Untreated/Zygote/08, Untreated/2-Cell/20, Untreated/8-Cell/02, NocMo/2-Cell/37, NocMo/8-Cell/12, MoNoc/2-Cell/22 and MoNoc/8-Cell/12	Untreated/Zygote/15, Untreated/2-Cell/06, Untreated/8-Cell/03, NocMo/2-Cell/01, NocMo/8-Cell/05, MoNoc/2-Cell/05 and MoNoc/8-Cell/02	Untreated/Zygote/04, Untreated/2-Cell/18, Untreated/8-Cell/01, NocMo/2-Cell/05, NocMo/8-Cell/11, MoNoc/2-Cel/12 and MoNoc/8-Cell/11
+Experiment Imaging Method	light sheet microscopy; confocal microscopy	light sheet microscopy	confocal microscopy	confocal microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	confocal microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	confocal microscopy	confocal microscopy	confocal microscopy
```

Comment:
This field and the one below should probably be reduced to 2 columns only: `light sheet microscopy` and `confocal microscopy` and the map annotation should have only 2 entries

Reply:
The problem is that the "Experiments" (Datasets) have different methods, some are light sheet, some are confocoal. So the second column is a summary which describes the "Experiment" (in the IDR sense) ('light sheet microscopy; confocal microscopy') and the other columns are specific to the dataset. I could reduce it to two columns but then loose the context of which dataset is which method.

Reply:
Well, actuall this information is also still available in the description. Ok. I'll reduce the number of columns, and also put the "Experiment" (Dataset) descriptions into the "Experiment Comments" field.